pkg/replicate/replicater.go

```diff
@@ -171,7 +171,7 @@ func RunReplicate(
 		level.Info(logger).Log("msg", "running replication attempt")
 
 		if err := newReplicationScheme(logger, metrics, blockFilter, fromBkt, toBkt, reg).execute(ctx); err != nil {
-			return errors.Errorf("replication execute: %w", err)
+			return errors.Errorf("replication execute: %v", err)
```

Comment:
Why don't we replace these with `errors.Wrap`?

Reply:
Actually, yes. This approach would be better. I haven't dug that into what they do while I was changing those. I'm gonna fix them.

Reply:
Yes, Wrap would be better ((: